app/serializers/api/variant_serializer.rb

```diff
@@ -30,4 +30,12 @@ def tag_list
 
     object.tag_list
   end
+
+  def thumb_url
+    if object.images.present?
```

Comment:
I just tried playing with this and it gives the same result either way, whether using `object.product.images.first` or `object.images.first`. :man_shrugging: 

I've updated it to use `object.product.images.first` because in reality we only save images on the product, right? So this is clearer.

Reply:
not exactly, we can save an image to a specific variant.
In the new image page there's a field called Version which is the variant.
![image](https://user-images.githubusercontent.com/1640378/83441653-c6e44e00-a43e-11ea-801c-cbb27ccebb3b.png)

But this is not used anywhere and should probably be removed. So Ithink it's ok to use the product image :+1: